demos/wasi/simpleserver/simpleserver.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: simpleserver
+  labels:
+    app: simpleserver
+spec:
+  containers:
+    - image: webassembly.azurecr.io/simpleserver:v1.0.0
+      imagePullPolicy: Always
+      name: simpleserver
+      ports:
+        - containerPort: 80
```

Comment:
I don't think you need this container port

Reply:
I copied the YAML straight from teh tutorial, so we should probably fix it there in a follow-up PR.